test/penalizer/Penalizer.test.ts

```diff
@@ -307,3 +365,44 @@ contract('Penalizer', function ([relayOwner, relayWorker, relayManager, otherAcc
     return [rr, sig]
   }
 })
+
+function getSecondsSinceUnixEpoch (): number {
+  return Math.floor(Date.now() / 1000)
+}
+
+async function assertTransactionFails (rawTx: string, reason: string): Promise<void> {
+  try {
+    await web3.eth.sendSignedTransaction(rawTx)
+    fail("expected claim to fail, but it didn't")
+  } catch (err) {
+    if (err instanceof Error) {
+      assert.isTrue(err.message.includes(reason), `unexpected revert reason: ${err.message}`)
+    } else {
+      fail('Unknown error')
+    }
+  }
+}
+
+async function assertClaimIsSuccessful (relayHub: RelayHubInstance, relayManager: string, sender: AccountKeypair, rawTx: string): Promise<void> {
```

Comment:
nice extraction :clap:

i know names are mostly personal preference, but aren't we actually checking that the penalization/stake burning was successful, instead of the claim being successful?

we know the claim was executed successfully before calling this, no? what we're doing here instead is checking that the funds staked for the manager were burned, to make sure the correct penalization—in terms of what funds are being taken from whom—took place in the hub contract.

anyways, not a blocker, sorry if i am being too petty with names :sweat_smile:

Reply:
I have Hamletic doubts each time I name something, so any suggestion is more than welcome! I think that something like `assertStakeIsBurned` would be a more appropriate description because it reflects better what the function checks for 👍🏼

Reply:
thanks, i think the new name is better :+1: